front/app/containers/Admin/reporting/components/ReportBuilder/Editor/index.tsx

```diff
@@ -27,7 +26,7 @@ const resolver = {
   ProjectTemplate,
   CommunityMonitorTemplate,
   PhaseTemplate,
-  Insights,
+  Insights: () => <div />,
```

Comment:
This needs to be taken out of the data via migration before it can be deleted to avoid crashing the report builder.